src/languages/en.ts

```diff
@@ -3452,7 +3452,7 @@ const translations = {
             somethingMightBeBroken: 'Or something might be broken. Either way, if you have any questions, just',
             contactConcierge: 'contact Concierge',
             chooseTransactionStartDate: 'Choose a transaction start date',
-            startDateDescription: 'We will import all transaction from this date onwards. If no date is specified, we’ll go as far back as your bank allows.',
+            startDateDescription: 'We'll import all transaction from this date onwards. If no date is specified, we’ll go as far back as your bank allows.',
```

Comment:
```suggestion
            startDateDescription: "We'll import all transaction from this date onwards. If no date is specified, we’ll go as far back as your bank allows.",
```

Hey @jamesdeanexpensify, when we use `'` in the middle of a text we have to use double quotes to wrap it so JavaScript doesn't get confused.

Reply:
Ah yep - thank you!